src/pykiso/test_coordinator/test_execution.py

```diff
@@ -199,20 +201,19 @@ def enable_step_report(all_tests_to_run: unittest.suite.TestSuite) -> None:
     # Step report header fed during test
     base_suite = test_suite.flatten(all_tests_to_run)
     for tc in base_suite:
-        # for any test, show ITF version
-        tc.step_report = StepReportData(
-            header=OrderedDict({"ITF version": pykiso.__version__})
-        )
-
-        # Decorate All assert method
-        assert_method_list = [
-            method for method in dir(tc) if method.startswith("assert")
-        ]
-        for method_name in assert_method_list:
-            # Get method from name
-            method = getattr(tc, method_name)
-            # Add decorator to the existing method
-            setattr(tc, method_name, assert_decorator(method))
+        tc.step_report = StepReportData()
```

Comment:
I forgot we got that here... I do not like it

Reply:
Only other possibility would be to add it by default in BasicTest, BasicTestSuiteSetup and BasicTestSuiteTearDown

Reply:
if we do that we have to redesign a little bit our framework

Reply:
sorry it makes two words

Reply:
Yea... I was thinking really to cluster it into the test_result section in the future.
and the part where we decorate the `asserts`, we make it a little bit nicer and independent of the step reporter.